components/InputTypeCountry.js

```diff
@@ -89,6 +89,7 @@ class InputTypeCountry extends Component {
     return (
       <StyledSelect
         name={name}
+        inputId="country-input"
```

Comment:
We should avoid doing that since there can be multiple country pickers on the page. It's up to the places that use InputTypeCountry to define an inputId.